modules/survey_accounts/js/survey_accounts_helper.js

```diff
@@ -39,7 +39,10 @@ $(document).ready(function () {
                 $('#create_survey').removeAttr('disabled');
                 $('#email_survey').attr('disabled','disabled');
             }
-
+            if (email.length > 0 && email2.length > 0 && email !== email2)
```

Comment:
Shouldnt this (and the one above) be

```
if ((email.length > 0 || email2.length > 0) && email !== email2)
```

?

Reply:
@driusan First condition(above of this) is to disable the Email Survey Button.  if (email2.length == 0 || email.length == 0 || email != email2) , disable Email Survey Button if any of the condition is true.

if (email.length > 0 && email2.length > 0 && email !== email2) -> this is for the correct error message to pop up when both email and confirm email have values and not matching.

Reply:
Oh, my bad. GitHub was hiding that part of the code unless I manually expanded it.